NTumbleBit/ClassicTumbler/ClassicTumblerParameters.cs

```diff
@@ -228,7 +225,8 @@ public bool IsStandard()
                 this.RealPuzzleCount == 15 &&
                 this.RealTransactionCount == 42 &&
                 this.FakeTransactionCount == 42 &&
-                (minExpectedFee < this.Fee && this.Fee < maxExpectedFee) &&
+                this.Denomination == new Money(0.01m, MoneyUnit.BTC) &&
```

Comment:
There seems to have been some sort of reversion here. Is this intentional?

Reply:
This is intentional